projects/igniteui-angular/src/lib/services/transaction/transaction.ts

```diff
@@ -26,6 +35,11 @@ export interface State {
     type: TransactionType;
 }
 
+export interface IStateUpdateEvent {
```

Comment:
we do not need `I` in front of the interface. In type script given class can `extend` other class or `implement` an interface. This is why when you read the class signature you know what class it extends and what interfaces it implements.